test/core.jl

```diff
@@ -4676,3 +4676,5 @@ if Sys.WORD_SIZE == 64
     end
     @test_nowarn tester20360()
 end
+
+eltype(Bottom) == Bottom
```

Comment:
fine, but it should be a `@test` instead of just returning false and moving on with no visible consequences if it ever happens to fail